tests/test_temporal.py

```
@@ -4686,6 +4686,67 @@ async def test_multimodal_content_serialization_in_workflow(client: Client):
+            args=['inspect attachment'],
+            id='test_nested_multimodal_tool_return',
+            task_queue=TASK_QUEUE,
+        )
+
+    tool_return = next(
+        part
+        for message in messages
+        for part in message.parts
+        if isinstance(part, ToolReturnPart) and part.tool_name == 'get_nested_multimodal_content'
+    )
+    tool_return_content_obj = getattr(tool_return, 'content')
```

Ruff (0.15.20)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)